src/backend/src/modules/pia-intake/pia-intake.controller.ts

```diff
@@ -184,7 +184,7 @@ export class PiaIntakeController {
     description:
       'Failed to update the PIA: The record is marked inactive in our system',
   })
-  @HttpCode(HttpStatus.OK)
+  @HttpCode(HttpStatus.NO_CONTENT)
```

Comment:
this api endpoint does not return any content to frontend, so change 200 to 204
the frontend http util function expect a json object for all non 204 code. otherwise it throw error
` if (response.status !== 204) {
        return response.json();
      }`